x509/x509.go

```diff
@@ -539,14 +540,22 @@ func getPublicKeyAlgorithmFromOID(oid asn1.ObjectIdentifier) PublicKeyAlgorithm
 // secp521r1 OBJECT IDENTIFIER ::= {
 //   iso(1) identified-organization(3) certicom(132) curve(0) 35 }
 //
+// secp192r1 OBJECT IDENTIFIER ::= {
```

Comment:
please modify the commit message s/p192r1/secp192r1/.
Searching the internet for `p192r1` returns several different curves.

Reply:
Done (with a rebase as a result).

BTW, RFC 5480 s2.1.1.1 has:

"Note that in [X9.62] the curves are referred to as 'ansiX9' as  opposed to 'sec'.  For example, secp192r1 is the same curve as ansix9p192r1.  Note that in [PKI-ALG] the secp192r1 curve was referred to as prime192v1 [...]  Note that [FIPS186-3] refers to secp192r1 as P-192 ..."

So there's multiple names for the same curve, and multiple curves with the same name.  Sigh.